database/schema.sql

```diff
@@ -42,6 +42,16 @@ CREATE INDEX agents_starttime_idx ON agents(starttime DESC);
 CREATE INDEX agents_queueloc_pid_idx ON agents(queueloc, pid);
 CREATE INDEX agents_status_idx ON agents(status);
 
+CREATE TABLE agent_action_relation (
```

Comment:
I think we aren't using this relationship anymore, right? @zsck

Reply:
Right you are. I've removed it.